examples/testscript_3d.py

```diff
@@ -138,11 +137,8 @@
     project_name = path_config_file.split(os.sep)[-2]
 
     os.chdir(os.path.join(project_name, "calibration_images"))
-    # Downloading the calibration images
-    url = "http://www.vision.caltech.edu/bouguetj/calib_doc/htmls/stereo_example.zip"
-    file_name = "stereo_example.zip"
-    with urllib.request.urlopen(url) as response, open(file_name, "wb") as out_file:
-        shutil.copyfileobj(response, out_file)
+
+    file_name = os.path.join(basepath,"stereo_example.zip")
```

Comment:
hi @jesse-gilmer dropping the download link on purpose?

Reply:
@MMathisLab I think it would be better to have a download link, but as far as I can tell the files needed are now embedded in another, much larger zip file (https://data.caltech.edu/records/jx9cx-fdh55) and it would take more work in the code to extract the stereo params files from caltech. The original link is dead. To use the fix as I have written it the user needs to download the zip manually, so I included it in the repo to reflect that. But, the file belongs to the caltech group so that probably isn't a usable solution.